src/shim/state_tree_v0.rs

```diff
@@ -0,0 +1,80 @@
+// Copyright 2019-2023 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+// ported from commit hash b622af
+
+// The FVM crates only support state tree versions 3,4 and 5. This module
+// contains read-only support for state tree version 0. This version is required
+// to parse genesis states. Ideally, we would have a library that supports _all_
```

Comment:
It's not a huge deal, but could we contrive some tests for this? We do have the genesis CAR after all.

Reply:
I added two tests for inspecting the state trees of the calibnet genesis block and mainnet genesis block.

Reply:
They're in the main `state_tree.rs` file.